### PR TITLE
Investment Game

### PR DESCRIPTION
**Pull request by andydrew93**
Not sure if I am doing this correctly

### CODE REVIEW

**Comment by markwhiting** [comment]
@andydrew93 — looks like #7 is ready for phase 2. 

@rafgehrke1 — could you give it a shot.

File: tasks/Investment Game.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# References
+> Berg, J., Dickhaut, J., & McCabe, K. (1995). Trust, Reciprocity, and Social History. Games and Economic Behavior, 10(1), 122–142.
+> https://econweb.ucsd.edu/~jandreon/Econ264/papers/Berg%20et%20al%20GEB%201995.pdf
+
+# Stimuli
+## The visual components
+> The experiment took place in person. Subjects were in each condition were in separate rooms. The subjects in room A were handed a large manilla envelope with a smaller envelop to place money in and a key corresponding to the room they were in. Subjects were then instructed to go behind a privacy partition and place the amount of money they wanted to “invest” in the envelope. The subject then places the large opaque envelope into the return box.
+
+
```
Comment (rafgehrke1) [inline tasks/Investment Game.md:12]: ## Materials for alternative versions of the experiment
> An alternative version of the experiment is to provide to the subjects a summary of previous interactions between rooms A and B as seen in the picture below (Add the chart) located at the top of page 141.

File: tasks/Investment Game.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Room B: 
+1 Participants receive triple the amount given by participant in room A
+Repeat steps 2-4 from room A
+
+## Rolls 
+> Subjects are assigned to either room A or B. Subjects in room A are given a 10 dollar ‘showing up fee’ and have to decide how much to send to subjects in room B. Room B participants receive tripled money and are asked to decide how much to send back to those in room A.
+
+# Criteria
+## Performance calculation
+> There is no correct answer. The amount of money being sent back and forth is the only thing being measured.
```
Comment (rafgehrke1) [inline tasks/Investment Game.md:29]: I would note that ... -> A metric evaluated by the source paper was the mean and deviation of the amount of money sent from both rooms A and B in the 2 versions of the experiment (supplying the history chart for the subjetcs in the beginning vs not supplying it)

**Comment by rafgehrke1** [comment]
The rendering of the .md seems to be missing some line breaks. For example, in the "steps" part the source code has line breaks like:
1 ....
2....
3...

but the rendering of the md looks like "1 ... 2 ... 3 ..."

**Review by rafgehrke1 — APPROVED** [review]
Good to go

**Pull request merged**